src/main/java/com/bankxpvalue/ItemDataCache.java

```diff
@@ -60,12 +65,13 @@ private void mapSkills(){
 
     // Stores json data in hashmap
     private void populateCache(){
-        final Gson gson = new Gson();
-        final InputStream itemData = ItemDataCache.class.getResourceAsStream("/item_xp_data.json");
-
-        ItemDataContainer data = gson.fromJson(new InputStreamReader(itemData, StandardCharsets.UTF_8), ItemDataContainer.class);
-        for (int i = 0; i < data.items.size(); i++) {
-            cache.put(data.items.get(i).id, data.items.get(i));
+        try (Reader reader = new InputStreamReader(getClass().getResourceAsStream("/item_xp_data.json"), StandardCharsets.UTF_8)) {
+            ItemDataContainer data = gson.fromJson(reader, ItemDataContainer.class);
```

Comment:
Throws null pointer on instantiation:

> Task :BankXpValuePluginTest.main()
2023-05-16 23:00:48 EDT [main] INFO  net.runelite.client.RuneLite - RuneLite 1.10.0.6 (launcher version unknown) starting up, args: none
2023-05-16 23:00:48 EDT [main] INFO  net.runelite.client.RuneLite - Java VM arguments: -Dfile.encoding=UTF-8 -Duser.country=US -Duser.language=en -Duser.variant -ea
2023-05-16 23:00:50 EDT [Preloader] INFO  net.runelite.client.rs.ClientLoader - Using cached vanilla client
2023-05-16 23:00:50 EDT [Preloader] INFO  net.runelite.client.rs.ClientLoader - Using cached patched client
2023-05-16 23:00:50 EDT [Preloader] INFO  net.runelite.client.rs.ClientLoader - client-patch 4985698149.135
2023-05-16 23:00:50 EDT [main] INFO  n.r.client.account.SessionManager - No session file exists
2023-05-16 23:00:50 EDT [main] INFO  n.r.client.config.ConfigManager - Profile 'default' (sync: false, active: true)
2023-05-16 23:00:50 EDT [main] INFO  n.r.client.config.ConfigManager - Using profile: default
2023-05-16 23:00:50 EDT [main] INFO  n.r.c.plugins.kourendlibrary.Library - Library is now reset
2023-05-16 23:00:51 EDT [main] INFO  n.r.c.e.ExternalPluginManager - Loading external plugin "117hd" version "1.2.8" commit "21215f65770453f3b27dfa0c932f2f27d5c221d8"
2023-05-16 23:00:51 EDT [main] WARN  n.r.client.plugins.PluginManager - Error instantiating plugin!
net.runelite.client.plugins.PluginInstantiationException: com.google.inject.CreationException: Unable to create injector, see the following errors:

```
 1) Error injecting constructor, java.lang.NullPointerException
  at com.bankxpvalue.ItemDataCache.<init>(Unknown Source)
  while locating com.bankxpvalue.ItemDataCache
    for field at com.bankxpvalue.BankXpValueItemOverlay.data(Unknown Source)
  while locating com.bankxpvalue.BankXpValueItemOverlay
    for field at com.bankxpvalue.BankXpValuePlugin.itemOverlay(Unknown Source)
  at net.runelite.client.plugins.PluginManager.lambda$instantiate$6(PluginManager.java:580)
Caused by: java.lang.NullPointerException
	at com.bankxpvalue.ItemDataCache.populateCache(ItemDataCache.java:69)
	at com.bankxpvalue.ItemDataCache.<init>(ItemDataCache.java:48)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at com.google.inject.internal.DefaultConstructionProxyFactory$ReflectiveProxy.newInstance(DefaultConstructionProxyFactory.java:126)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:111)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:90)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:268)
	at com.google.inject.internal.SingleFieldInjector.inject(SingleFieldInjector.java:54)
	at com.google.inject.internal.MembersInjectorImpl.injectMembers(MembersInjectorImpl.java:132)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:120)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:90)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:268)
	at com.google.inject.internal.SingleFieldInjector.inject(SingleFieldInjector.java:54)
	at com.google.inject.internal.MembersInjectorImpl.injectMembers(MembersInjectorImpl.java:132)
	at com.google.inject.internal.MembersInjectorImpl$1.call(MembersInjectorImpl.java:93)
	at com.google.inject.internal.MembersInjectorImpl$1.call(MembersInjectorImpl.java:80)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1085)
	at com.google.inject.internal.MembersInjectorImpl.injectAndNotify(MembersInjectorImpl.java:80)
	at com.google.inject.internal.Initializer$InjectableReference.get(Initializer.java:223)
	at com.google.inject.internal.Initializer.injectAll(Initializer.java:132)
	at com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:174)
	at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:110)
	at com.google.inject.internal.InjectorImpl.createChildInjector(InjectorImpl.java:232)
	at com.google.inject.internal.InjectorImpl.createChildInjector(InjectorImpl.java:236)
	at net.runelite.client.plugins.PluginManager.instantiate(PluginManager.java:583)
Caused by: java.lang.NullPointerException

	at net.runelite.client.plugins.PluginManager.loadPlugins(PluginManager.java:392)
	at net.runelite.client.externalplugins.ExternalPluginManager.loadExternalPlugins(ExternalPluginManager.java:122)
	at net.runelite.client.RuneLite.start(RuneLite.java:353)
	at net.runelite.client.RuneLite.main(RuneLite.java:282)
	at com.bankxpvalue.BankXpValuePluginTest.main(BankXpValuePluginTest.java:11)

2) Error injecting constructor, java.lang.NullPointerException
  at com.bankxpvalue.ItemDataCache.<init>(Unknown Source)
  while locating com.bankxpvalue.ItemDataCache
    for field at com.bankxpvalue.BankXpValuePlugin.data(Unknown Source)
  at net.runelite.client.plugins.PluginManager.lambda$instantiate$6(PluginManager.java:580)
Caused by: java.lang.NullPointerException
	at com.bankxpvalue.ItemDataCache.populateCache(ItemDataCache.java:69)
	at com.bankxpvalue.ItemDataCache.<init>(ItemDataCache.java:48)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at com.google.inject.internal.DefaultConstructionProxyFactory$ReflectiveProxy.newInstance(DefaultConstructionProxyFactory.java:126)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:111)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:90)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:268)
	at com.google.inject.internal.SingleFieldInjector.inject(SingleFieldInjector.java:54)
	at com.google.inject.internal.MembersInjectorImpl.injectMembers(MembersInjectorImpl.java:132)
	at com.google.inject.internal.MembersInjectorImpl$1.call(MembersInjectorImpl.java:93)
	at com.google.inject.internal.MembersInjectorImpl$1.call(MembersInjectorImpl.java:80)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1085)
	at com.google.inject.internal.MembersInjectorImpl.injectAndNotify(MembersInjectorImpl.java:80)
	at com.google.inject.internal.Initializer$InjectableReference.get(Initializer.java:223)
	at com.google.inject.internal.Initializer.injectAll(Initializer.java:132)
	at com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:174)
	at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:110)
	at com.google.inject.internal.InjectorImpl.createChildInjector(InjectorImpl.java:232)
	at com.google.inject.internal.InjectorImpl.createChildInjector(InjectorImpl.java:236)
	at net.runelite.client.plugins.PluginManager.instantiate(PluginManager.java:583)
	at net.runelite.client.plugins.PluginManager.loadPlugins(PluginManager.java:392)
	at net.runelite.client.externalplugins.ExternalPluginManager.loadExternalPlugins(ExternalPluginManager.java:122)
	at net.runelite.client.RuneLite.start(RuneLite.java:353)
	at net.runelite.client.RuneLite.main(RuneLite.java:282)
	at com.bankxpvalue.BankXpValuePluginTest.main(BankXpValuePluginTest.java:11)
```